src/components/input-field/input-field.tsx

```diff
@@ -204,6 +214,7 @@ export class InputField {
                     {this.label}
                 </span>
                 {this.renderTrailingIcon()}
+                {this.renderTrailingErrorIcon()}
```

Comment:
It should not be possible to have both the regular icon and the error icon. If there is an error, the error icon should replace the regular icon